src/app/asset-page/asset-page.component.pug

```diff
@@ -177,8 +177,8 @@
             ng-container(*ngIf="fieldName != 'Rights'")
               .label([attr.id]="cleanId(fieldName)") {{ fieldName }}
               div(*ngFor="let value of assets[0].formattedMetadata[fieldName]; let i=index")
-                a.value.col-link(*ngIf="(fieldName === 'Collection' && i===0)", (click)="trackCollectionLink(cleanFieldValue(value))", [routerLink]="setCollectionLink(assets[0])", [innerHTML]="cleanFieldValue(value) | linkify", [attr.aria-labelledby]="cleanId(fieldName)")
-                .value(*ngIf="(fieldName !== 'Collection' || i!==0)", [innerHTML]="cleanFieldValue(value) | linkify", [attr.aria-labelledby]="cleanId(fieldName)")
```

Comment:
The check here for NOT index 0 prevents double display of collection links. Now that we want to display more than one possible Collection link, Line# 181 needs a check that compares the values of multiple collections to remove duplicates.